src/test/java/edu/cornell/kfs/rass/batch/service/impl/RassServiceImplTest.java

```diff
@@ -911,5 +911,50 @@ private ExpectedProcessingResults expectedResults(
     private interface ObjectUpdateValidator<T, U> {
         void assertObjectWasUpdatedAsExpected(T expectedObject, U actualObject, int index);
     }
+    
+    @Test
+    public void testSortFileNames() {
+        String a = "a";
+        String b = "b";
+        String c = "c";
+        List<String> results = buildFileNameList(c, b, a);
+        
+        assertEquals(a, results.get(2));
+        
+        rassService.sortFileNameList(results);
+        
+        assertEquals(a, results.get(0));
+        assertEquals(b, results.get(1));
+        assertEquals(c, results.get(2));
+    }
+    
+    @Test
+    public void testSortFIleNamesWithRealLifeFileNames() {
```

Comment:
Minor typo in method name; should be changed from "...SortFIleNames..." to "...SortFileNames..."